docs/spec/sanitations.md

```diff
@@ -12,16 +12,10 @@ These changes are done in order to improve the overall usability, and as workaro
 
 1. Remove resource paths,
     */playlists/{playlist_id}/images'(put) - schema for the image/jpeg content type is incorrectly defined
-2.  Remove inline response schemas.
-     - headers - Headers to be sent with the request 
-     - queries - Queries to be sent with the request
-3.  Remove bad responses to increase readability.
-        401':
-          $ref: '#/components/responses/Unauthorized'
-        '403':
-          $ref: '#/components/responses/Forbidden'
-        '429':
-          $ref: '#/components/responses/TooManyRequests'
```

Comment:
@Sachindu-Nethmin we need to keep the sanitation number 3 (Remove bad responses to increase readability) so don't revert it. 

We only need to do what @aashikam mentioned in https://github.com/ballerina-platform/module-ballerinax-spotify/pull/2#issuecomment-2435443820, instead of removing header and query parameters